test/SourceKit/CursorInfo/cursor_implicit_init.swift

```diff
@@ -11,8 +11,12 @@ func foo() {
   MySubClass(abc:1)
 }
 
-// RUN: %sourcekitd-test -req=cursor -pos=10:12 %s -- %s | %FileCheck -check-prefix=CHECK1 %s
-// CHECK1: <Declaration>init(abc: <Type usr="s:Si">Int</Type>)</Declaration>
+// RUN: %sourcekitd-test -req=cursor -pos=10:7 %s -- %s | %FileCheck -check-prefix=CHECK1-INIT %s
+// RUN: %sourcekitd-test -req=cursor -pos=10:12 %s -- %s | %FileCheck -check-prefix=CHECK1-PAR %s
+// CHECK1-INIT: <Declaration>init(abc: <Type usr="s:Si">Int</Type>)</Declaration>
+// CHECK1-PAR: <Declaration>let abc: <Type usr="s:Si">Int</Type></Declaration>
```

Comment:
This is a change of behavior. Previously, invoking jump-to-defintion on a parameter of an initializer took you to the initializer (bypassing the disambiguation of whether you wanted to jump to the type or initializer). If we decide that the change of behavior is correct, I would like to have a `CHECK` line here that ensures we get a source location back for the parameter, which will ensure that we jump to the parameter’s definition.

CC @bnbarham. You might have opinions on this as well.